ReactWindows/ReactNative/Modules/StatusBar/StatusBarModule.cs

```diff
@@ -0,0 +1,227 @@
+﻿using ReactNative.Bridge;
+using ReactNative.UIManager;
+using System;
+using Windows.Foundation.Metadata;
+using Windows.UI;
+using Windows.UI.Core;
+using Windows.ApplicationModel.Core;
+
+namespace ReactNative.Modules.StatusBar
+{
+    /// <summary>
+    /// A module that allows JS to set statusbar properties.
+    /// </summary>
+    public class StatusBarModule : NativeModuleBase
+    {
+        /// <summary>
+        /// Runtime platform.
+        /// </summary>
+        public enum PlatformType
+        {
+            /// <summary>
+            /// Mobile.
+            /// </summary>
+            Mobile,
+            /// <summary>
+            /// Desktop.
+            /// </summary>
+            Desktop,
+            /// <summary>
+            /// Other.
+            /// </summary>
+            Other,
+        };
+
+        private PlatformType _platformType;
+        private IStatusBar _statusBar;
+        private ITitleBar _titleBar;

```

Comment:
It will significantly reduce the number of lines of code.

---
In reply to: [57043831](https://github.com/CatalystCode/react-native/pull/274#discussion_r57043831) [](ancestors = 57043831)